core/templates/pages/exploration-editor-page/editor-tab/graph-directives/state-graph-visualization.component.ts

```diff
@@ -451,15 +453,14 @@ export class StateGraphVisualization
       this.nodeList.push(this.nodeData[nodeId]);
     }
 
-    this.overallTransformStr = 'translate(0,0)';
-    this.innerTransformStr = 'translate(0,0)';
-
     if (this.allowPanning) {
       this.makeGraphPannable();
     }
 
     if (this.centerAtCurrentState) {
-      this.centerGraph();
+      setTimeout(() => {
+        this.centerGraph();
+      });
```

Comment:
Can we please add a comment to explain why setTimeout is required?  Also is default value for setTimeout ( which is 0 ms) sufficient even in slow connections?

Reply:
Yes, a 0 value is sufficient as we only want to run this code once all the current call stack is finished executing.